### PR TITLE
Issue #2472815 doExecute for DataConvert

### PR DESCRIPTION
**Pull request by botris**
Updated the execute() function to doExecute() for the DataConvert class.
Issue #2472815

### CODE REVIEW

File: src/Plugin/RulesAction/DataConvert.php [hunk 1]
```diff
@@ -43,13 +43,15 @@ class DataConvert extends RulesActionBase {
 
   /**
    * Executes the plugin.
+   *
+   * @param $value
```
Comment (klausi) [inline src/Plugin/RulesAction/DataConvert.php:47]: data type missing, see https://www.drupal.org/coding-standards/docs#param

Comment (klausi) [inline src/Plugin/RulesAction/DataConvert.php:47]: If it can be anything we can use "mixed"

**Comment by ndeet** [comment]
I took over, we should close this. https://github.com/fago/rules/pull/304

**Comment by klausi** [comment]
closing.

**Pull request closed without merging**